integr8ly/integreatly-operator
# MGDAPI-5877

# Issue link
[MGDPI-5877](https://issues.redhat.com/browse/MGDAPI-5877)

# What
Available condition type of  `MonitoringStack` is now reported in `AddonInstance`

# Verification steps
1. Checkout this branch
2.  Run 
``` 
make cluster/prepare/local
make code/run
```
3. Wait for RHOAM to install
4.  `AddonInstance` should now also report the availability of `MonitoringStack`
```
watch "oc get addoninstance addon-instance -n redhat-rhoam-operator -o json | jq '.status'
```
Expected Results
```
conditions:
    - lastTransitionTime: '2023-08-04T13:58:21Z'
      message: Heartbeat not received before timeout threshold.
      observedGeneration: 1
      reason: HeartbeatTimeout
      status: Unknown
      type: addons.managed.openshift.io/Healthy
    - lastTransitionTime: '2023-08-04T12:20:47Z'
      message: Installation complete
      observedGeneration: 1
      reason: SetupComplete
      status: 'True'
      type: addons.managed.openshift.io/Installed
    - lastTransitionTime: '2023-08-04T12:23:47Z'
      message: Core components healthy
      observedGeneration: 1
      reason: Healthy
      status: 'True'
      type: integreatly.org/Healthy
    - lastTransitionTime: '2023-08-04T12:23:47Z'
      message: All components healthy
      observedGeneration: 1
      reason: Degraded
      status: 'False'
      type: addons.managed.openshift.io/Degraded
    - lastTransitionTime: '2023-08-04T12:19:25Z'
      message: Monitoring Stack is available
      observedGeneration: 1
      reason: MonitoringStackAvailable
      status: 'True'
      type: MonitoringStackAvailable
  lastHeartbeatTime: '2023-08-04T13:57:51Z'
  observedGeneration: 1
```
